product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.1-converting-soap-to-json/1.1.1-soap-to-json-using-payloadfactory-mediator/pom.xml

```diff
@@ -32,7 +32,59 @@
         <plugins>
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <inherited>true</inherited>
+                <configuration>
```

Comment:
Should we need this configuration since it is already defined in the parent pom?